Funçao de inicializaçao de um usuario

rodolfotr opened the pull request:
#1 Adicionado funcionalidade de iniciar topico de controle do usuario

Commit:
Funçao de inicializaçao de um usuario

Adicionado funcionalidade de iniciar topico de controle do usuario

renanbabinski commented on main.c:
```diff
@@ -43,47 +46,99 @@ int list_menu(){
 }
+
+int initializeUser(MQTTClient conn, MQTTClient_connectOptions opts, MQTTClient_willOptions wopts, char* userID){
+	int client = 0;
```
@rodolfotr 
aqui dá pra deixar:
`int client;`

renanbabinski commented on main.c:
```diff
@@ -43,47 +46,99 @@ int list_menu(){
+	opts.MQTTVersion = MQTTVERSION_DEFAULT;
+	opts.username = userID;
+
+	opts.will = &wopts;
+	opts.will->message = userID;
+	opts.will->qos = 1;
+	opts.will->retained = 0;
+	opts.will->topicName = userTopic;
+	opts.will = NULL;
```
@rodolfotr 
`opts.will = NULL`
Acaba limpando as definições das linha anteriores.

renanbabinski reviewed (changes requested):
@rodolfotr só veja esses pontos que comentei se fazem sentido pra ti

rodolfotr committed:
Ajustes Solicitados

Ajustado conforme solicitado.
Inicialização da variável Client.
Preenchimento da estrutura Opts->Will.
